src/org/usfirst/frc/team694/robot/commands/AcquirerAcquireCommand.java

```diff
@@ -5,7 +5,7 @@
 import edu.wpi.first.wpilibj.command.Command;
 
 /**
- *
+ * Turn the Acquirer Motors on
```

Comment:
Try to be a bit more specific, turning them on doesn't really tell us what it does, nor it's purpose.